README.md

```diff
@@ -120,6 +122,17 @@ You can skip the email authentication locally or in some of the testing environm
 
 3. If you do both, then the app will always use what you've set in the url parameter as the first responder's email.
 
+## Save and return feature <a name="save-and-return-feature"></a>
+
+You can develop and test the save and return feature on you local machine. You will need to set up the [save-and-return-api](https://github.com/UKHomeOffice/save-return-api) so it is running on your local machine.
+
+You can then start the server in the api development mode.
+
+```bash
+$ yarn dev:api
+```
+To skip the email verification follow the [Skip email verify step](#skip-email-verify-step) instructions. 
```

Comment:
you probably don't need this line as we have mentioned it and it's added in the contents section :)

Reply:
Okay I'll remove this.